test/test_library.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// @file
+/// Trivial library to ensure we have some linking present for
+/// `test_find_library`.
+
+namespace test_library
+{
+
+RCPPUTILS_PUBLIC
```

Comment:
This needs an `#include` of the header containing the macro definition.

Please rebuild the package locally and run tests when making changes.

Reply:
Ah... oops! Done.